### DISCUSSION TITLE
針對某個特定輸入方案設定豎排顯示候選詞

### DISCUSSION THREAD

**Discussion by xty** [rime/home#1771] [Q&A] [answered]
我在`ibus_rime.custom.yaml`檔案中設定了橫排顯示候選詞：
```yaml
patch:
  style/horizontal: true
```
但是[粵拼（jyut6ping3）](https://github.com/rime/rime-cantonese)的候選詞很長，所以我希望在粵拼方案下使用豎排（覆寫全域設定）。請問應該怎樣設定？

**Comment by lotem** [comment, marked as answer]
不支持的。
